app/assets/javascripts/components/CustomLink.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const CustomLink = React.createClass({
```

Comment:
A top-level comment explaining the purpose of this component would be helpful, since it has a very generic name. It should point to which circumstances you want to use this instead of the non-custom `Link`.

Reply:
have added a comment `Replacement for non-custom Link component. The react Link component doesn't highlight the active link if it is not handeled by react router, CustomLink is used to provide that functioning.`